lib/Onnxifi/Base.cpp

```diff
@@ -168,16 +166,30 @@ onnxStatus Graph::setIOAndRun(uint32_t inputsCount,
 
     auto &outPhPtr = outPhIt->getValue();
 
-    phNameToOnnxTensorOutputs[outPhPtr] = outOnnxTensor;
+    // Compute the total size of the onnxifi tensor.
+    std::vector<size_t> outOnnxTensorDims(outOnnxTensor.dimensions);
```

Comment:
Minor, but I believe we can skip building this vector by constructing an ArrayRef from `outOnnxTensor.shape, outOnnxTensor.dimensions`.

We can probably then get the total size via std::accumulate over that ArrayRef but that might not end up clearer.

Reply:
Eh ok I just tried this out and I think it looks more confusing, I'm just going to keep it like it is matching the inputs code